docs/DeveloperGuide.md

```diff
@@ -274,7 +274,13 @@ _{Explain here how the data archiving feature will be implemented}_
 
 **Target user profile**:
 
-* has a need to manage a significant number of contacts
+* administrator of a small Non-Governmental Organisation (NGO) 
```

Comment:
Perhaps small should be defined more clearly? E.g. in terms of revenue or number of full-time employees etc.

Reply:
thank you minlong for the suggestion! indeed i do think i can define it more clearly. i will make the necessary changes 👍